the-blue-alliance-ios/View Controllers/Teams/TeamMediaCollectionViewController.swift

```diff
@@ -94,16 +95,11 @@ class TeamMediaCollectionViewController: TBACollectionViewController {
     // MARK: UICollectionView Delegate
     
     override func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
-        // TODO: Eventually show the full image inside the app
-        guard let media = dataSource?.object(at: indexPath) else {
-            return
-        }
-        guard let url = media.viewImageURL else {
-            return
-        }
-        if UIApplication.shared.canOpenURL(url) {
-            UIApplication.shared.open(url, options: [:], completionHandler: nil)
-        }
+        let imageViewer = Optik.imageViewer(
```

Comment:
If you're viewing a team with multiple images and click on an image, we don't open up the right image. This always shows the first image